source/index.erb

```diff
@@ -2,36 +2,53 @@
 title: Energy Globe Award
 ---
 
-<div id="application">
-
-  <header>
-    <h1 class="headline">Energy Globe Award</h1>
-  </header>
-
-  <section ng-controller="AwardsController">
-
-    <section class="filters">
-      <h2>Kategorien</h2>
-      <ul class="category-filter" ng-repeat="category in filters.category.names">
-        <li>
-          <input type="checkbox" id="category_checkbox_{{category}}" ng-model="filters.category.filterMap[category]">
-          <label for="category_checkbox_{{category}}">{{category}}</label>
-        </li>
-      </ul>
-
-      <h2>Jahre</h2>
-      <ul class="year-filter" ng-repeat="year in filters.year.names">
-        <li>
-          <input type="checkbox" id="year_checkbox_{{year}}" ng-model="filters.year.filterMap[year]">
-          <label for="year_checkbox_{{year}}">{{year}}</label>
-        </li>
-      <ul>
-    </section>
-
-    <div class="award" ng-repeat="award in filteredAwards">
-      <h3>{{ award.title }}</h3>
+<header class="page-header">
+  <div class="page-header-container">
+    <div class="page-header-content">
+      <h1 class="page-header-headline">Energy Globe Award</h1>
```

Comment:
The problem is, it doesn't prevent collisions, it's probably actually more prone to them. What you did is just spreading the scope everywhere into each sub element. its still totally possible that i write a `.category-filter-headline` style somewhere in the top level and accidentally override your stuff.

The scoping style makes it clearer in my opinion that you should never define *general* classes like `content` or `container` at the top level. Every part of the page should be enclosed in some semantic scope like `awards-result-list` which can also be an `id` because we won't have two of them on any page.